### PR TITLE
elastic-stack

### PR DESCRIPTION
**Pull request by AyadiAmen**
Elastic-stack feature for the logs management.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [ ] Variables are documented in the README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,9 +65,12 @@ Each requirement is configured with the options provided by that Chart. Please c
...
 | `phpldapadmin.enabled`                                                      | Enable phpLDAPadmin                                                                                                | `true`                          |
-| `elasticstack.enabled`                                                      | Enable Elastic-stack                                                                                               | `false`                         |
 | `kafka.enabled`                                                             | Enable Kafka                                                                                                       | `false`                         |
 | `cassandra.enabled`                                                         | Enable Cassandra                                                                                                   | `false`                         |
+| `filebeat.enabled`                                                         | Enable Filebeat                                                                                                   | `false`                         |
```
Comment (alexnuttinck) [inline README.md:70]: @AyadiAmen what is the purpose of filebeat? As ELK = Elasticksearch, Logstash, Kibana

File: values.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 spark:
-  enabled: true
+  enabled: false
```
Comment (alexnuttinck) [inline values.yaml]: @AyadiAmen you have disabled all the services, please enable them and disable the elk-stack.

File: templates/NOTES.txt [hunk 3]
```diff
@@ -305,28 +305,77 @@ Test server health using Helm test:
...
+    {{- else if contains "ClusterIP"  .Values.kibana.service.type }}
+
+    export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app={{ template "kibana.name" . }},release={{ .Release.Name }}" -o jsonpath="{.items[0].metadata.name}")
+    echo "Visit http://127.0.0.1:5601 to use Kibana"
+    kubectl port-forward --namespace {{ .Release.Namespace }} $POD_NAME 5601:5601
+    {{- end }}
 
 {{- end }}
 
+
```
Comment (alexnuttinck) [inline templates/NOTES.txt:393]: @AyadiAmen, There are no notes about elasticsearch, is it wanted?

**Review by alexnuttinck — CHANGES_REQUESTED** [review]
@AyadiAmen, there are some issues to do before merging. See my comments.

File: README.md (hunk 1 above)
Comment (AyadiAmen) [reply README.md:70]: @alexnuttinck Filebeat is used to easily ship log file data to Logstash and Elasticsearch to centralize your logs and analyze them in real time, Logstash is useful if you need to  apply some common transformations and filtering to your events.

**Comment by AyadiAmen** [comment]
@alexnuttinck thank you for your review, I will fix the issues as soon as I can.

File: README.md (hunk 1 above)
Comment (alexnuttinck) [reply README.md:70]: @AyadiAmen ok, thanks for the explanation.

**Review by alexnuttinck — APPROVED** [review]
It seems ok, thanks for the modifications.

**Pull request merged**